feat(docs/ulule-api): change main_image fields to i18n objects (#89)
ulule/developers.ulule.com: content/docs/ulule-api/reference/reward.md

| --------------------- | ----------------------------------------------------------- | -------------------------------------------------------------------------------------------------------- |
| `available`           | bool                                                        | Whether the reward is available                                                                          |
| `delivery`            | [delivery resource](#delivery-resource)                     | Delivery settings for the reward, copied from project delivery settings, or custom for the reward        |
| `description`         | [i18n object](#i18n)                                        | Description of the reward, can contain HTML tags                                                         |
| `has_custom_delivery` | bool                                                        | Whether the reward has custom delivery settings                                                          |
| `id`                  | int                                                         | Unique ID of the reward                                                                                  |

> | `image`               | [i18n object](#i18n) with [image resource](#image) as value | Image of the reward                                                                                      |

Reviewer: ```suggestion
| `image`               | [i18n object](#i18n) where values are [image resources](#image) | Image of the reward                                                                                      |
```
